find_sentence.py

```diff
@@ -0,0 +1,49 @@
+
+def find_intersection(dict1, dict2):
```

Comment:
Could you please provide some documentation for this code? It will help reviewers understand the purpose and functionality of the function.